src/docs/environment.mdx

```diff
@@ -164,3 +164,122 @@ If you would like to be able to run `devserver` outside of your root checkout, y
   <em>and</em> port are required in order for DNS and some pages urls to be
   displayed correctly.
 </Alert>
+
+
+## Troubleshooting
+
+**Problem:** something like `pkg_resources.DistributionNotFound: The 'some_dependency<0.6.0,>=0.5.5' distribution was not found and is required by sentry`
+
+**Solution:** Your virtualenv needs to be updated. In sentry, type `make install-sentry-dev`. In getsentry, type `make install-develop-requirements`.
```

Comment:
note to self: I'm gonna unify those two commands to just be `install-dev`. There's no reason they should be different.

Reply:
https://github.com/getsentry/sentry/pull/19017
https://github.com/getsentry/getsentry/pull/3944

Will resolve this when you make the edits.

Reply:
Fixed now.